src/main/resources/emails/emailReport.ftl

```diff
@@ -469,8 +469,8 @@ ${sender.name} sent you a report from ANET:
     </div>
   </#if>
 
-  <#if report.loadApprovalStatus()??>
-    <#assign approvalStatus = report.loadApprovalStatus()>
+  <#if report.loadApprovalStatus(context)??>
+    <#assign approvalStatus = report.loadApprovalStatus(context)>
```

Comment:
this is doing the same again, shouldn't we define a getApprovalStatus (like it has been done with loadLocation and getLocation for instance?) and use it in here?

Reply:
agreed, will push in 10

Reply:
hmm, I think we should get rid of the getXXX / loadXXX combo eventually, so I made a single call on the ftl side instead.